tailwind.config.ts

```diff
@@ -65,7 +65,16 @@ export default {
   	lg: 'var(--radius)',
   	md: 'calc(var(--radius) - 2px)',
   	sm: 'calc(var(--radius) - 4px)'
-      }
+      },
+	  keyframes: {
```

Comment:
doesn't look like this is aligned, i think there's an extra tab before 'keyframes'